app/models/ingredient.rb

```diff
@@ -0,0 +1,7 @@
+class Ingredient < ApplicationRecord
+  validates_presence_of :name
```

Comment:
I know the base repo still has some validations, but just keep in mind that it's not required to add validations on an IC! :)